sources/us/co/sanmiguel.json

```diff
@@ -17,13 +17,14 @@
     "conform": {
         "number": "STREETNO",
         "street": [
-          "STREETDIR",
-          "STREETNAME",
-          "STREETSUF",
-          "STREETPOD"
+            "STREETDIR",
+            "STREETNAME",
+            "STREETSUF",
+            "STREETPOD"
         ],
         "unit": "STREETALP",
         "city": "LOC_CITY",
-        "type": "shapefile-polygon"
+        "type": "shapefile-polygon",
+        "postcode": "zipcode"
```

Comment:
Same here, `zipcode` refers to the owner mailing address, not the situs physical zip code.